packages/components/avatar/src/avatar.styles.ts

```diff
@@ -0,0 +1,93 @@
+import { designTokens } from '@commercetools-uikit/design-system';
+import { css } from '@emotion/react';
+import startCase from 'lodash/startCase';
+import { TAvatarProps } from './avatar';
+
+const fontSizeMap = {
+  s: {
+    narrow: designTokens.fontSize30, // '16px',
+    wide: designTokens.fontSize10, // '12px',
+  },
+  m: {
+    narrow: designTokens.fontSize50, // '20px',
+    wide: designTokens.fontSize30, // '16px',
+  },
+  l: {
+    narrow: designTokens.fontSize90, //'48px',
+    wide: designTokens.fontSize80, // '40px',
+  },
```

Comment:
I left those commented pixels I used in my branch we should remove them. 🙏

Reply:
I'll do this now